documentation/assemblies/overview/assembly-kafka-components.adoc

```diff
@@ -0,0 +1,23 @@
+// This assembly is included in:
+//
+// overview/master.adoc
+
+[id="kafka-components_{context}"]
+= {ProductName} deployment of Kafka
+
+Apache Kafka components are provided for deployment to Kubernetes with the {ProductName} distribution.
+The Kafka components are generally run as clusters for availability.
+
+A typical deployment incorporating Kafka components might include:
+
+* *Kafka* cluster of broker nodes
+* *ZooKeeper* cluster of replicated ZooKeeper instances
+* *Kafka Connect* cluster for external data connections
+* *Kafka MirrorMaker* cluster to mirror the Kafka cluster in a secondary cluster
+* *Kafka Exporter* to extract Kafka metrics data for monitoring
```

Comment:
in this way it seems that you NEED Kafka Exporter for having Kafka metrics while we know that it's not mandatory. Kafka Exporter provides additional metrics (like consumer lag) compared to the ones we have just using the Kafka custom resource (and the related `metrics` field + Prometheus/Grafana deployment).

Reply:
It says _typical_ deployment. 
Maybe something like after the list: 
_Not all of these components are mandatory, though you need Kafka and ZooKeeper as a minimum._

Reply:
@ppatierno - is this okay? :point_up_2:

Reply:
@PaulRMellor the problem is that the sentence says "... to extract Kafka metrics data for monitoring" which sounds to me "in order to have metrics you NEED to deploy Kafka Exporter". As I said it's not true. Without Kafka Exporter you already have metrics exported by Kafka and Zookeeper. The Exporter component allows to have "additional" metrics.

Reply:
As discussed, I added 'additional' to the Exporter sentence.